src/libraries/System.Reflection.Emit/tests/ModuleBuilder/ModuleBuilderDefineType.cs

```diff
@@ -37,7 +37,7 @@ public static IEnumerable<object[]> TestData()
         [MemberData(nameof(TestData))]
         public void DefineType(string name, TypeAttributes attributes, Type parent, PackingSize packingSize, int typesize, Type[] implementedInterfaces)
         {
-            bool isDefaultImplementedInterfaces = implementedInterfaces?.Length == 0;
+            bool isDefaultImplementedInterfaces = implementedInterfaces != null && implementedInterfaces.Length == 0;
```

Comment:
The behavior is explicit and well known for these cases with `!= null && Length == 0`. We are not interested in changing them into more verbose form.